library/src/main/java/com/codetroopers/betterpickers/radialtimepicker/RadialTimePickerDialog.java

```diff
@@ -516,7 +516,7 @@ private void setCurrentItemShowing(int index, boolean animateCircle, boolean del
      * @return true if the key was successfully processed, false otherwise.
      */
     private boolean processKeyUp(int keyCode) {
-        if (keyCode == KeyEvent.KEYCODE_ESCAPE || keyCode == KeyEvent.KEYCODE_BACK) {
```

Comment:
Ok, 
So lets remove this from this PR and create a new one. So I could discuss with teammate what to do with this and merge to part about hour recurrence